php/meterpreter/meterpreter.php

```diff
@@ -630,8 +655,16 @@ function rand_xor_byte() {
     return chr(mt_rand(1, 255));
```

Comment:
Do we need to? According to https://stackoverflow.com/questions/34318732/when-will-php-generate-the-seed-for-mt-rand it is automatically seeded the first time it is called in a script.